src/Controls/tests/Core.UnitTests/BrushUnitTests.cs

```diff
@@ -84,7 +84,6 @@ public void TestBrushParent()
 
 			parent.Background = linearGradientBrush;
 
-			Assert.Same(parent, parent.Background.Parent);
 			Assert.Same(context, parent.Background.BindingContext);
```

Comment:
This is a behavior change that we don't set `Background.Parent` anymore.

However, `Background.BindingContext` is set, so you should be able to use `{Binding}` such as:

```xml
<Label>
  <Label.Background>
    <SolidColorBrush Color="{Binding Foo}" />
  </Label.Background>
</Label>
```

And the `Label`'s `BindingContext` should pass on to the `SolidColorBrush`.

Reply:
Yeah, this was always interesting as a single brush can be used for multiple views. Is there anything that needs access to the brush/stroke's parent? If there is, then this is really interesting as who wins? Last one? First one?

Reply:
I am also wondering about that binding context... If I set one brush on 2 views, is it just going to overwrite the BC to the last one? Maybe that is just how it works?